### PR TITLE
Program february

### PR DESCRIPTION
**Pull request by TueHenriksen**
Please review this suggestion of the program.

@PhilipK , we need the address of the venue, and if possible a bar where we can drink a beer afterwards.

@skovmand , will you be the host for yet another book lottery?

@GeeWee and @mikkelhegn, please review your sections. I have linked to your GitHub profiles. Feel free to change it if you prefer to link elsewhere (e.g., LinkedIn). Let me know if you need to change the timing a bit when we get closer and you have prepared your talk.

### CODE REVIEW

**Review by GeeWee — APPROVED** [review]
Looks good to me!

File: meetups/2024/2024-02-22-talks/program.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## Zero-copy deserialization in Rust
+By [Gustav Wengel][gw]
+
+When using Rust in many contexts, such as webservers or just parsing structured data from files, we need to deserialize text or bytes into Rust types. The de-facto library for this in the Rust ecosystem is [serde] - and we're going to explore how serde allows us to deserialize objects without actually allocating memory, by borrowing from the original source. We'll also dive into the [rkyv] crate as a zero-copy deserialization library if you really need every last bit of performance.
+
+Gustav has been working with Rust for the past two years, building serverless RESTful API's at the startup Climatiq, that allows the users to embed CO2 and climate calculations into existing software. Before that he has worked in different startups with different tech stacks, and has been a core maintainer of popular open-source projects like [ts-jest].
+
+
+## Location
+This meetup will be at [INSERT ADDRESS HERE]. Please feel free to use the event chat on meetup to organize transportation if needed.
```
Comment (PhilipK) [inline meetups/2024/2024-02-22-talks/program.md:36]: I am pretty sure the address is, but I am awaiting confirmation from Partisia:
INCUBA Katrinebjerg, Lille Auditorium
Åbogade 15
8200 Aarhus

Comment (TueHenriksen) [reply meetups/2024/2024-02-22-talks/program.md:36]: ok, I will wait for the final confirmation

**Comment by mikkelhegn** [comment]
LGTM 👍🏻

**Review by mikkelhegn — APPROVED** [review]

File: meetups/2024/2024-02-22-talks/program.md (hunk 1 above)
Comment (PhilipK) [reply meetups/2024/2024-02-22-talks/program.md:36]: Confirmed, The address is correct.

**Comment by TueHenriksen** [comment]
> @TueHenriksen When do you expect this program to be announced? Would love to be able to start promoting it.

I had hoped that it was already ready to be published, but it sounds like we need to squeeze in one more talk. I really hope we will have something at the end of this week.

File: meetups/2024/2024-02-22-talks/program.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+The program for the evening is:
+
+- 18:05 - 18:10: Welcome (5 min)
+- 18:15 - 19:00: "Building serverless apps, using WASI, WebAssembly components using Rust" by Mikkel Mørk Hegnhøj (45 min incl Q&A)
+- 19:00 - 19:10: A word from our sponsor (5-10 min)
+- 19:10 - 19:55: Light dinner sponsored by Partisia (45 min)
+- 19:55 - 20:40: "Zero-copy deserialization in Rust" by Gustav Wengel (45 min incl Q&A)
+- 20:45 - 20:55: Book Lottery (10 min)
+- 20:55 - 21:00: Closing remarks (5 min)
+- 21:00: Let's continue the discussion at the nearby bar [INSERT BAR NAME].
```
Comment (PhilipK) [inline meetups/2024/2024-02-22-talks/program.md:16]: Lets omit this part about the bar for now, so we can get this published, then we can always update it later to add the bar.
```suggestion
```

File: meetups/2024/2024-02-22-talks/program.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Date: 2024-02-22
+
+Rust Aarhus invites you to this free meetup for all interested in the programming language [Rust].
+
+The program for the evening is:
+
+- 18:05 - 18:10: Welcome (5 min)
+- 18:15 - 19:00: "Building serverless apps, using WASI, WebAssembly components using Rust" by Mikkel Mørk Hegnhøj (45 min incl Q&A)
+- 19:00 - 19:10: A word from our sponsor (5-10 min)
+- 19:10 - 19:55: Light dinner sponsored by Partisia (45 min)
```
Comment (PhilipK) [inline meetups/2024/2024-02-22-talks/program.md:12]: Partisia is still figuring out who does their talk, so I think we should just add this, and we can update the agenda later with the actual speaker, if we want (for me it is also fine to just let it be as the company name).
I took 10 minutes from the lunch time, we still have 35 mins, which should be plenty time to eat a sandwich.
```suggestion
- 19:10 - 19:20: Using Rust at Partisia by Partisia (10 min)
- 19:20 - 19:55: Light dinner sponsored by Partisia (45 min)
```

**Review by PhilipK — APPROVED** [review]
A few suggestions to get this thing out there, so we can start promoting it.

File: meetups/2024/2024-02-22-talks/program.md (hunk 1 above)
Comment (TueHenriksen) [inline meetups/2024/2024-02-22-talks/program.md:36]:
```suggestion
This meetup will be at INCUBA Katrinebjerg, Lille Auditorium, Åbogade 15, 8200 Aarhus. Please feel free to use the event chat on meetup to organize transportation if needed.
```

**Pull request merged**